### ISSUE TITLE
[RFC] Organize Scene Filter Dropdown

### ISSUE THREAD

**Issue by jeremymeyers** [stashapp/stash#1545]
<!-- Update or delete the title if you need to delegate your title gore to something
# Title

*### Scope*
Refactoring the dropdown list for the scene filter

##  Long Form
![Screenshot 2021-07-02 01 05 28](https://user-images.githubusercontent.com/3514095/124223471-c6026d80-dad1-11eb-81af-21a496cedec9.png)
![Screenshot 2021-07-02 01 05 25](https://user-images.githubusercontent.com/3514095/124223472-c6026d80-dad1-11eb-831b-218ba0f3c285.png)

Suggested Arrangement with separators between  (and headers as disabled) (order within sections not final)

FILE INFO
- Path
- Size
- Last Modified
- Created
- Frame Rate
- Bit Rate

SCENE INFO
- Title
- Date
- Duration
- Interactive
- Rating
- Performer Count
- Tag Count

DATABASE
- Scene Number
- O-Counter
- Organized
- Last Updated

**Comment by CosimoVestri** [comment]
I think this is a good idea, but think that the with the given items the order should be "SCENE INFO", "DATABASE", "FILE INFO."

For most users, I would say that FILE INFO is the least used sorting criteria.

Perhaps O-Counter should be with Scene Info. This is similar information to rating, thus being in same group makes more sense to me.

**Comment by jeremymeyers** [comment]
Yup, it was not intended to depict specific order, just categories

**Comment by DogmaDragon** [comment]
Using something like [dropdown headers](https://bootstrapshuffle.com/classes/dropdowns/dro) would provide more separation and clarity.
